src/common/settings.js

```diff
@@ -547,6 +547,12 @@ const character_settings = {
         "type": "bool",
         "default": false
     },
+    "ranger-natural-explorer": {
+        "title": "Ranger: Natural Explorer",
+        "description": "Your familiarity with certain regions aids in your exploration",
```

Comment:
I would change this into "Your familiarity with the current region" so it's clear that it's meant to boost the user rather than just say "I have this feature" (which is a mistake some users will inevitably make I think).
Not a blocker though, thx!